realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -888,6 +914,47 @@ void BaseRealSenseNode::setupPublishers()
     }
 }
 
+void BaseRealSenseNode::addMonitoredTopic(const stream_index_pair& stream, Topic topic, const std::string& name)
+{
+    const std::lock_guard<std::mutex> lock(_topic_monitor_mutex);
+    TopicInfo info;
```

Comment:
The TopicInfo definition and topic monitoring method definitions appear to be missing from the header file.